recipes/panda3d_viewer/meta.yaml

```diff
@@ -0,0 +1,52 @@
+{% set name = "panda3d_viewer" %}
+{% set version = "0.4.1" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/panda3d_viewer-{{ version }}.tar.gz
+  sha256: 623842303b9f568d80f5a300d35079738f578917916394abaca03ac4fa7c130a
+
+build:
+  number: 0
+  skip: True  # [win]
+  skip: True  # [py<36]
+  # The following skip will be removed in the feedstock
+  skip: True  # [py>36]
+  script: {{ PYTHON }} -m pip install . -vv
+
+requirements:
+  host:
+    - python
+    - pip
+  run:
+    - python
+    - panda3d
```

Comment:
Be aware that when submitting two recipes that and one depends on the other you are creating a race condition on the feedstock creation and `panda3d_viewer` will most likely fail. Do you want to send `panda3d` first and then this one?

Reply:
What is a race condition exactly ?

Reply:
Both feedstocks will be created at the same time so the one that depends on the other will fail due to a missing dependency.

Reply:
Ok,  should I split in two PR ?
However, since, I have the skip remove in pand3d anyway, I can let the feedstock create and race. PR + merge the skip remove on panda3d and when it is online, pr + merge the skip remove in panda3d_viewer.

Reply:
If you are OK with the latter, sure. Some users get lost when a package is not created.

Reply:
No problem with the later, I already maitain some recipes depending on each other